docs/sources/source/conf.py

```diff
@@ -89,10 +98,10 @@
 #
 # The short X.Y version.
 # version = "0.10.0"
-version = str(nemo.__version__)
+version = "0.10"  # str(nemo.__version__)
```

Comment:
why?

Reply:
I explained both those  during our conversation two days ago - both are temporary solutions that enable us to generate few documentation for EN for `master` and `0.10.0` - this is my short-term goal.

This one on to inject version this way you need to "import nemo", which in turns requires to install all NeMo dependencies. Currently I simply do not know how to solve it now in other way. Any help here?